.github/CODEOWNERS

```diff
@@ -29,4 +32,6 @@
 /Userland/Utilities/js.cpp @linusg
 /Userland/Utilities/sql.cpp @trflynn89
 /Userland/Utilities/wasm.cpp @alimpfard
+/Toolchain @bgianfo
+/Tests @bgianfo
```

Comment:
Please keep these in alphabetical order :redlinus:

Reply:
Also just to double check, are you sure you want to be on /Tests? That'll notify you anytime someone touches any unit test ever